nix/devtools.nix

```diff
@@ -96,6 +97,29 @@ let
         ${style}/bin/postgrest-lint
         ${style}/bin/postgrest-style-check
       '';
+
+  importsgraph =
+    checkedShellScript
+      {
+        name = "postgrest-importsgraph";
+        docs =
+          ''
+            Render the imports between PostgREST modules as a graph.
+
+            Output is written to 'imports.png' in the root directory of the project.
+          '';
+        inRootDir = true;
+      }
+      ''
+        grep -rE 'import .*PostgREST\.' src \
```

Comment:
You might want to have a look at GHC's `-ddump-minimal-imports` option. This should return stuff in a better to parse way. Parsing the output of that could also allow us to add more data about those imports to the graph. You could e.g. add the number of functions/types used in this import on the edges of the graph?

Reply:
To get the dump, I ran:

```bash
mkdir dump
cabal v2-build --ghc-option=-ddump-minimal-imports --ghc-option=-dumpdir=dump
```

Example output: https://gist.github.com/monacoremo/d72726e7afad5c05f95cc34301c62a9f

Suggest that we keep the current approach for now, as parsing the number of imports could be tricky and running the build to generate the dump adds a long extra step. But we should keep it in mind for later improvements!

Reply:
@wolfgangwalther Experimented on https://github.com/monacoremo/postgrest/tree/symbolimports

This yields a map of all imported symbols:
![symbols](https://user-images.githubusercontent.com/59358383/107154505-86e14d80-6973-11eb-9a7b-446cff91ccae.png)

Many symbols are only imported from one module, might be good leads for refactoring!